src/main/java/DukeTest.java

```diff
@@ -0,0 +1,108 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class DukeTest {
+    private static ArrayList<Task> taskList = new ArrayList<>();
+
+    private static boolean isExit(String input) {
+        return input.equals("bye");
+    }
+
+    private static void exit() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    private static void addTask(Task task) {
+        taskList.add(task);
+        System.out.println("Got it. I've added this task:\n" + task.toString() +
+                "\nNow you have " + taskList.size() + " tasks in the list.");
+    }
+
+    private static void deleteTask(int idx) {
+        Task temp = taskList.get(idx - 1);
+        taskList.remove(idx - 1);
+        System.out.println("Noted. I've removed this task:\n" + temp.toString() +
+                "\nNow you have " + taskList.size() + " tasks in the list.");
+
+    }
+
+    private static void list() {
+        for (int i = 1; i <= taskList.size(); i++) {
+            System.out.println(i + "." + taskList.get(i - 1).toString());
+        }
+    }
+
+    public static void main(String[] args) {
+        try {
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+
+            String input = scanner.nextLine();
+
+            String[] splitInput = input.split(" ", 2);
+            ;
+            String command = splitInput[0];
+
+            while (!(isExit(command))) {
+                switch (command) {
```

Comment:
Great that you used switch case and have modularized your code to perform the respective actions for the different conditions, looks neat!